### ISSUE TITLE
form elements do not have associated labels

### ISSUE THREAD

**Issue by a20behta** [HGustavs/LenaSYS#11790]
related to issue #9800 
using for controls makes it easier for different screen reader programs to work as intended. double check if it is possible to change/give correct lable. if so change the correct label. 

check if it is caused by autogenerated html or hard coded. 
issue appears on different sites so expect autogenerated.

**Comment by b20jonlu** [comment]
![form](https://user-images.githubusercontent.com/81630271/165046088-33e50387-6bd5-42c6-9484-347d938254b1.png)

**Comment by b20jonlu** [comment]
### Sectioned.php

All these checkboxes are missing labels, the problem is that it is quite difficult to add labels to them without making the site look worse,
![these checkboxes are missing labels](https://user-images.githubusercontent.com/102533453/165255935-f0c23dff-2164-487e-8940-132712d0f3f1.png)

This is what it would look like if we added a label to each checkbox that just states the name it belongs to. A shorter name would look better (also had to make the space belonging to the checkbox bigger to fit text otherwise a row takes up a lot more vertical space).
![something like this](https://user-images.githubusercontent.com/102533453/165256661-095a63a0-dd23-4ecd-8211-df0f55afbbf7.png)

One of the solutions i thought about was making the text on the left (for example "PHP Example 2") a label the problem with that is that clicking the label is the same as clicking the checkbox and the text itself is already a link to another page. When clicking a checkbox that item is added to an array so if we click something like the "PHP examples" header we also add it to the array which probably isn't what we want. 

I am in need of suggestions on how to tackle this problem since i feel no solution i have thought of is quite right.

**Comment by b20jonlu** [comment]
### Sectioned.php

All these checkboxes are missing labels, the problem is that it is quite difficult to add labels to them without making the site look worse,
![these checkboxes are missing labels](https://user-images.githubusercontent.com/102533453/165255935-f0c23dff-2164-487e-8940-132712d0f3f1.png)

This is what it would look like if we added a label to each checkbox that just states the name it belongs to. A shorter name would look better (also had to make the space belonging to the checkbox bigger to fit text otherwise a row takes up a lot more vertical space).
![something like this](https://user-images.githubusercontent.com/102533453/165256661-095a63a0-dd23-4ecd-8211-df0f55afbbf7.png)

One of the solutions i thought about was making the text on the left (for example "PHP Example 2") a label the problem with that is that clicking the label is the same as clicking the checkbox and the text itself is already a link to another page. When clicking a checkbox that item is added to an array so if we click something like the "PHP examples" header we also add it to the array which probably isn't what we want. 

I am in need of suggestions on how to tackle this problem since i feel no solution i have thought of is quite right.

**Comment by a20behta** [comment]
can the text/label be hidden. the main reason is for screen readers and other accessability tools, so if it is in the html it should improve the results directly. 

@b20jonlu 
how would that look

**Comment by b20jonlu** [comment]
Yeah i think i found a solution to that just a moment ago when checking. There is a tag that can be used called placeholder which is used on for example input type search. This would be hidden when applied to a checkbox so it would probably work.
![Screenshot (20)](https://user-images.githubusercontent.com/102533453/165263741-4799750a-72f1-45a0-850b-e9271a2d2381.png)
This method does not give a warning when using lighthouse so i am going with that aproach. What do you think the text inside the placeholder should be?
